askama/src/filters/mod.rs

```diff
@@ -7,23 +7,23 @@
 
 use std::fmt::{self, Write};
 
-#[cfg(feature = "serde_json")]
+#[cfg(feature = "serde-json")]
 mod json;
-#[cfg(feature = "serde_json")]
+#[cfg(feature = "serde-json")]
 pub use self::json::json;
 
-#[cfg(feature = "serde_yaml")]
+#[cfg(feature = "serde-yaml")]
 mod yaml;
-#[cfg(feature = "serde_yaml")]
+#[cfg(feature = "serde-yaml")]
 pub use self::yaml::yaml;
 
 #[allow(unused_imports)]
 use crate::error::Error::Fmt;
 use askama_escape::{Escaper, MarkupDisplay};
 #[cfg(feature = "humansize")]
-use humansize::{file_size_opts, FileSize};
+use mod_humansize::{file_size_opts, FileSize};
 #[cfg(feature = "num-traits")]
-use num_traits::{cast::NumCast, Signed};
+use mod_num_traits::{cast::NumCast, Signed};
```

Comment:
A `mod_` prefix is better than a `_` suffix, I guess. :) I'm open for other ideas.

Reply:
There is also the possibility of having overlapping module / external crate names and disambiguating using `::humansize` for the crate and `self::humansize` for the module.

Reply:
But the problem here isn't module vs crate names but crate name vs feature name, IIRC?

Reply:
That's what the other review thread is about, this looks separate to me.

Reply:
No, it's the same issue.

Reply:
Oh, so `mod_` as a crate name prefix? Yeah I think that's confusing, I would prefer `crate` if there is going to be a prefix (or suffix).

Reply:
To me a `crate` {pre,suf}fix sound like it's an alias for `crate::`. How about `pkg_*`, since the Cargo.toml option is called `package`?